src/main/java/org/broadinstitute/consent/http/service/ElectionService.java

```diff
@@ -426,6 +430,10 @@ public List<Election> createDataSetElections(String referenceId, Map<User, List<
         return CollectionUtils.isEmpty(electionsIds) ? null : electionDAO.findElectionsByIds(electionsIds);
     }
 
+    public List<Election> findElectionsByVoteIdsAndType(List<Integer> voteIds, String electionType) {
+        return electionDAO.findElectionsByVoteIdsAndType(voteIds, electionType);
```

Comment:
Usage of this look pretty safe, but it might be nice to add an empty list check here to ensure we never make the dao call.